YourToDo/templates/planner/planner.html

```diff
@@ -8,180 +8,171 @@
                     <div><input type="radio" name="layout" value="day" id="planner-day"><label for="planner-day">Day</label></div>
-                    <div><input type="radio" name="layout" value="week" checked="checked" id="planner-week"><label for="planner-week">Week</label></div>
-                    <div><input type="radio" name="layout" value="month" id="planner-month"><label for="planner-month">Month</label></div>
+                    <div><input type="radio" name="layout" value="week" id="planner-week"><label for="planner-week">Week</label></div>
+                    <div><input type="radio" name="layout" value="month" checked="checked" id="planner-month"><label for="planner-month">Month</label></div>
                 </td>
             </tr>
-            
-            <!-- Days List -->
-            <tr class="planner-week-days">
-                <td>Sunday</td>
-                <td>Monday</td>
-                <td>Tuesday</td>
-                <td>Wednesday</td>
-                <td>Thurday</td>
-                <td>Friday</td>
-                <td>Saturday</td>
-            </tr><!-- End Top Content -->
         </table>
 
         <!-- TODO: Add custom scrollbars that don't look awful -->
 
-        <div class="planner-week-content">
+        <div class="planner-month-content">
             <table class="planner-container">
                 <!-- Column Specification: Left sidebar, 7 days -->
-                <col class="planner-week-left-sidebar">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-days-day">
+                <col class="planner-month-right-sidebar">
 
-                <tr class="planner-week-row">
-                    <td class="planner-week-category-container btn-primary">
-                        EECS 448
-                    </td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                </tr>
-                <tr class="planner-week-row">
-                    <td class="planner-week-category-container btn-success">
-                        EECS 645
+                <tr class="planner-month-row">
+                    <td class="date"><h1>1</h1></td>
```

Comment:
Instead of applying a class "date" to each table data, why not just create a style

.planner-month-row td {
...
}